### DISCUSSION TITLE
About the new changes

### DISCUSSION THREAD

**Discussion by alemscoder** [GitbookIO/community#506] [2. Published content]
### Ask a question, start a conversation, or make an annoucement

In my case, in the edition mode still remain the old style but in preview have the new style, its normal? And the other problem is that the changes are not reflecting in the published link

**Comment by addisonschultz** [comment]
Hi @alemscoder - good catch. Right now, the updates only affect the published version. The editor style's we will be looking at updating next. I don't have an exact timeline on this, but it's definitely on our radar
